cranelift/filetests/filetests/isa/riscv64/simd-vconst-64bit.clif

```diff
@@ -0,0 +1,114 @@
+test compile precise-output
+set unwind_info=false
+target riscv64 has_v
+
+function %vconst_zeroes() -> i8x8 {
+block0:
+    v0 = vconst.i8x8 0x00
+    return v0
+}
+
+; VCode:
+; block0:
+;   vle8.v v10,[const(0)] #avl=8, #vtype=(e8, m1, ta, ma)
+;   ret
+; 
+; Disassembled:
+; block0: ; offset 0x0
+;   .byte 0x57, 0x70, 0x04, 0xcc
+;   auipc t6, 0
+;   addi t6, t6, 0x14
+;   .byte 0x07, 0x85, 0x0f, 0x02
+;   ret
+;   .byte 0x00, 0x00, 0x00, 0x00
+;   .byte 0x00, 0x00, 0x00, 0x00
+;   .byte 0x00, 0x00, 0x00, 0x00
+
+function %vconst_ones() -> i8x8 {
+block0:
+    v0 = vconst.i8x8 0xffffffffffffffff
+    return v0
+}
+
+; VCode:
+; block0:
+;   vle8.v v10,[const(0)] #avl=8, #vtype=(e8, m1, ta, ma)
+;   ret
+; 
+; Disassembled:
+; block0: ; offset 0x0
+;   .byte 0x57, 0x70, 0x04, 0xcc
+;   auipc t6, 0
+;   addi t6, t6, 0x14
+;   .byte 0x07, 0x85, 0x0f, 0x02
+;   ret
```

Comment:
It looks like we're discovering some gaps in capstone :(

Reply:
Yeah, unfortunately capstone doesn't recognize any V extension instructions 😞